kolibri/plugins/learn/assets/src/composables/useCopies.js

```diff
@@ -0,0 +1,21 @@
+import { set } from '@vueuse/core';
+import { reactive } from 'kolibri.lib.vueCompositionApi';
+
+// Shared store - an object because it uses Vue.observable under the hood
+var displayedCopies = reactive({ copies: [] });
```

Comment:
As I mentioned on Slack when we chatted about other composable, I think it'd be fine to use a shared global state in some composables when there's a good reason to do so. However, I don't understand why it's needed for this data? From code and clicking through the library, it seems to me copies show only in the copies modal that is open for one resource at a time. Was there any reason why having it defined in the function scope of `useCopies` wouldn't be sufficient?

Reply:
It looks like the global state is needed because this is referenced in the parent component that has the `CopiesModal` in it, and the child `CopiesModal` component.

I think using the composable in this case is adding an additional layer of indirection here - rather than simply passing in the copies via a prop, we are setting it in some global state, then reading from that global state inside another component.

Reply:
Ah okay, I haven't noticed this. In that case, it'd make sense to stay with props to me, too.